orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/tasks/PageApplicationOwnerTask.groovy

```diff
@@ -42,32 +44,54 @@ class PageApplicationOwnerTask implements RetryableTask {
   @Autowired(required = false)
   Front50Service front50Service
 
+  private final ObjectMapper mapper = new ObjectMapper()
```

Comment:
Again, should be able to `@Autowired` this in here _but_ I'm not sure you even need it. Doesn't appear to be used anywhere.

Reply:
Yep, leftover from a refactor. Good call.